src/Microsoft.Azure.CosmosRepository/Repositories/DefaultRepository.Specs.cs

```diff
@@ -19,9 +19,11 @@ internal sealed partial class DefaultRepository<TItem>
         QueryRequestOptions options = new();
 
         if (specification.UseContinuationToken)
-        {
             options.MaxItemCount = specification.PageSize;
-        }
+        
+        if(specification.PartitionKey != null)
```

Comment:
Prefer `null` constant pattern-matching expressions.

```suggestion
        if (specification.PartitionKey is not null)
```